install_config/aggregate_logging.adoc

```diff
@@ -441,7 +441,10 @@ openshift_logging_es_pvc_dynamic value.
 |`openshift_logging_es_pvc_size`
 |Size of the persistent volume claim to
 create per Elasticsearch instance. For example, 100G. If omitted, no PVCs are
-created and ephemeral volumes are used instead.
+created and ephemeral volumes are used instead. Also specify `openshift_logging_elasticsearch_storage_type=pvc`.
+
+|`openshift_logging_elasticsearch_storage_type`
+a|Sets the Elasticsearch storage type. If you are using xref:aggregated-logging-persistent-storage[Persistent Elasticsearch Storage], set to `pvc`.
```

Comment:
drop preceeding "a"

Reply:
The `a` is used in a table cell to preserve special formatting. I had some special formatting in the cell, but I don't think it is needed now. Removed.